### PR TITLE
Update to be compatible with Python 3

### PR DESCRIPTION
**Pull request by mitchute**

### CODE REVIEW

**Comment by mitchute** [comment]
@Myoldmopar there is still one test failing at this point, and it has gotten significantly slower. I suspect it's related to how you are reading in the files (byte buffer vs. str buffer).

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,29 @@
 language: python
```
Comment (Myoldmopar) [inline .travis.yml:1]: Could you add a small matrix in here to test both Python 2 and 3?

Comment (mitchute) [reply .travis.yml:1]: See if that's right

Comment (Myoldmopar) [reply .travis.yml:1]: Looks right but Travis will be quick to tell us otherwise :game_die:

**Comment by Myoldmopar** [comment]
Looks like one failed job is just a TOX_ENV vs TOX_EVN typo.  The other one looks like a few more str vs bytes changes between Python 2 and 3.  Let me know if you want me to tackle them.

**Comment by mitchute** [comment]
Let me work on it a little more. If I can't get it wrapped up pretty quick, I'll pass it over to you.

**Comment by mitchute** [comment]
Ready for review :)

**Pull request closed without merging**